src/libs/utils.test.ts

```diff
@@ -136,3 +136,27 @@ describe('isValidWsExportTarget', () => {
     expect(result).toBe(false)
   })
 })
+
+describe('formatBytes', () => {
+  it('handles GB', () => {
+    // Act
+    const result = formatBytes(40000000000)
+
+    // Assert
+    expect(result).toBe('37.3 GiB')
```

Comment:
what's up with the little 'i' in GiB?  This looks pretty awkward for the other results like MiB below...